docs/Oracle Jobs/jobs.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: nodes.liquid
+date: Last Modified
+title: "Jobs"
+permalink: "docs/jobs/"
+whatsnext: {"Tasks":"/docs/tasks/"}
+---
+
+## What is a Job?
+
+NOTE: This page refers to the latest version of Chainlink jobs (otherwise known as TOML, or V2 jobs). For documentation on the legacy job format, see [V1 job specs](/docs/job-specifications).
+
+Chainlink nodes require jobs to be added to do anything useful, for example, generating verifiable randomness or posting asset price data on-chain. There are various job types, including:
+
+- `cron`
+- `directrequest`
+- `fluxmonitor`
+- `keeper`
+- `offchainreporting`
+- `randomnesslog`
+
+Jobs are represented by TOML specifications. Examples for each type can be found below.
+
+---
+
+## Job types
+
+### Shared fields
+
+- `name`: the name of the job in the Operator UI.
+- `type`: specifies the v2 job type. Can be one of the following:
+    - `cron`
+    - `directrequest`
+    - `fluxmonitor`
+    - `keeper`
+    - `offchainreporting`
+    - `randomnesslog`
+- `schemaVersion`: not currently used, but must be present and set to a value of `1`. Will allow us to handle progressive iterations of the job spec format gracefully with backwards-compatibility.
+- `observationSource`: the v2 pipeline task DAG. Specified in DOT syntax. See below for information on writing pipeline DAGs.
+- `maxTaskDuration`: the default maximum duration that any given task is allowed to run. If the duration is exceeded, the task is errored. This value can be overridden per-task (see subsequent sections).
+
+### Cron
+
+Executes a job on a schedule. Does not rely on any kind of external trigger.
+
+```toml
+type            = "cron"
+schemaVersion   = 1
+schedule        = "0 0 0 1 1 *"
+observationSource   = """
+ds          [type=http method=GET url="https://chain.link/ETH-USD"];
+ds_parse    [type=jsonparse path="data,price"];
+ds_multiply [type=multiply times=100];
+ds -> ds_parse -> ds_multiply;
+"""
+```
+
+- `schedule`: the frequency with which the job is to be run, specified in traditional UNIX cron format (but with 6 fields, not 5. The extra field allows for "seconds" granularity). You can also use e.g. `@every 1h` if you just want something executed on a regular interval, but the wall clock time is not important.
+
+For all supported schedules, please refer to the [cron library documentation](https://pkg.go.dev/github.com/robfig/cron?utm_source=godoc).
+
+### Direct request (eth log)
+
```

Comment:
What's the replacement for RunLog in a new version?

Reply:
This job type replaces both runlog and ethlog. There's a discussion around whether we rename it to ethlog as thats just a better/more descriptive name than directrequest